<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCBTest</class>
 <widget class="QWidget" name="PCBTest">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>1144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>9000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PCBTest</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="MatplotlibWidgetScope" name="ScopeDisplay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>667</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>4</horstretch>
     <verstretch>3</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="MatplotlibWidgetWFPot" name="WaveformDisplay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>667</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>530</width>
     <height>400</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>STIXGeneral</family>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>650</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QSpinBox" name="inp_initVel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>700</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="inp_finalVel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>700</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="inp_aveSweeps_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Intial velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Final velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Intial vel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>140</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chk_plotWF">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
        </font>
       </property>
       <property name="text">
        <string>Plot WF</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_wfOutput">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
        </font>
       </property>
       <property name="text">
        <string>Start PCB Pot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_extTrig">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
        </font>
       </property>
       <property name="text">
        <string>ExtTrig WF</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>281</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>153</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="inp_aveSweeps">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>AveSweeps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
        </font>
       </property>
       <property name="text">
        <string>Save Trace</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>ScopeDisplay</zorder>
   <zorder>inp_aveSweeps</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>1031</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>138</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scope control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>620</y>
     <width>265</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Waveform potential control</string>
   </property>
  </widget>
  <zorder>ScopeDisplay</zorder>
  <zorder>WaveformDisplay</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>line</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidgetWFPot</class>
   <extends>QGraphicsView</extends>
   <header>MatplotlibWidgetWFPot</header>
  </customwidget>
  <customwidget>
   <class>MatplotlibWidgetScope</class>
   <extends>QGraphicsView</extends>
   <header>MatplotlibWidgetScope.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
